swarm/storage/ldbstore.go

```diff
@@ -602,41 +602,76 @@ func (s *LDBStore) CleanGCIndex() error {
 		return err
 	}
 
-	it.Seek([]byte{keyIndex})
-	var idx dpaDBIndex
+	it.Release()
+
+	//var idx dpaDBIndex
 	var poPtrs [256]uint64
-	for it.Valid() {
-		rowType, chunkHash := parseGCIdxKey(it.Key())
-		if rowType != keyIndex {
-			break
+	var doneIterating bool
+	for !doneIterating {
```

Comment:
We should add a `log.Debug` line here to use as a progress indicator when we run this.